packages/web/components/PageHeader.tsx

```diff
@@ -93,6 +103,15 @@ const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
 export const PageHeader: React.FC = () => {
   const { isOpen, onToggle, onClose } = useDisclosure();
 
+  const { connectWeb3, disconnect, isConnected } = useContext(Web3Context);
```

Comment:
There's a `useWeb3` hook in web/lib/hooks.js. you can use that instead of this

Reply:
It isn't necessary from now because `Web3Context` had been moved to `@metafam/utils`